openbanking/deploy/setup-ce.sh

```diff
@@ -0,0 +1,1040 @@
+                "labels": []
+            },
+            {
+                "name": "given_name",
+                "description": "Given name",
+                "type": "string",
+                "labels": [
+                    "simple"
+                ]
+            },
+            {
+                "name": "middle_name",
+                "description": "Middle name",
+                "type": "string",
+                "labels": []
+            },
+            {
+                "name": "family_name",
+                "description": "Family name",
+                "type": "string",
+                "labels": [
+                    "simple"
+                ]
+            },
+            {
+                "name": "nickname",
+                "description": "Nickname",
+                "type": "string",
+                "labels": []
+            },
+            {
+                "name": "preferred_username",
+                "description": "The primary username that represents the user",
+                "type": "string",
+                "labels": [
+                    "simple"
+                ]
+            },
+            {
+                "name": "profile",
+                "description": "URL of the profile page",
+                "type": "string",
+                "labels": []
+            },
+            {
+                "name": "picture",
+                "description": "URL of the profile picture",
+                "type": "string",
+                "labels": []
+            },
+            {
+                "name": "website",
+                "description": "URL of the web page",
+                "type": "string",
+                "labels": []
+            },
+            {
+                "name": "gender",
+                "description": "Gender",
+                "type": "string",
+                "labels": []
+            },
+            {
+                "name": "birthdate",
+                "description": "Birthdate",
+                "type": "string",
+                "labels": []
+            },
+            {
+                "name": "zoneinfo",
+                "description": "Zoneinfo",
+                "type": "string",
+                "labels": []
+            },
+            {
+                "name": "locale",
+                "description": "Locale",
+                "type": "string",
+                "labels": []
+            },
+            {
+                "name": "updated_at",
+                "description": "Last update",
+                "type": "number",
+                "labels": []
+            },
+            {
+                "name": "idp_sub",
+                "description": "IDP Subject",
+                "type": "string",
+                "labels": []
+            },
+            {
+                "name": "customer_id",
+                "description": "Customer ID",
+                "type": "string",
+                "labels": []
+            }
+        ]
+    },
```

Comment:
We now inject the `customer_id` attribute into cdr workspaces by default, so I think we can probably remove this.

Reply:
This value is hardcoded in the demo/tutorial and is used by apigeex. We dont do client registration so this is part of the mocked data.